### PR TITLE
fix(docs): fix broken links

### PR DESCRIPTION
**Pull request by wlee221**
#### Description of changes

- removes link to [https://css.gg/](https://css.gg/), which is broken right now
- update link to `Next.js` getting started, which has changed this morning.

#### Description of how you validated changes
ci

#### Checklist

- [x] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [x] PR description included
- [x] Relevant documentation is changed or added (and PR referenced)
- [x] `yarn test` passes and tests are updated/added
- [x] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/src/pages/[platform]/getting-started/usage/nextjs/react.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { InstallScripts, TerminalCommand } from '@/components/InstallScripts.tsx
 
 ## Tutorial
 
-In this brief tutorial, we're going to build a basic shopping card component using [Next.js](https://nextjs.org/docs/getting-started) and Amplify UI. Then, we'll elaborate on our shopping card to play with the possibilities of using Amplify UI.
+In this brief tutorial, we're going to build a basic shopping card component using [Next.js](https://nextjs.org/docs/getting-started/installation) and Amplify UI. Then, we'll elaborate on our shopping card to play with the possibilities of using Amplify UI.
```
Comment (0618) [inline docs/src/pages/[platform]/getting-started/usage/nextjs/react.mdx:8]: Or this link? https://nextjs.org/docs

**Review by 0618 — DISMISSED** [review]
LGTM! left a question

**Review by dbanksdesign — APPROVED** [review]

File: docs/src/pages/[platform]/getting-started/usage/nextjs/react.mdx (hunk 1 above)
Comment (wlee221) [reply docs/src/pages/[platform]/getting-started/usage/nextjs/react.mdx:8]: Oh yeah, that's better. Thanks!
```suggestion
In this brief tutorial, we're going to build a basic shopping card component using [Next.js](https://nextjs.org/docs) and Amplify UI. Then, we'll elaborate on our shopping card to play with the possibilities of using Amplify UI.
```

**Review by 0618 — APPROVED** [review]
🚢

**Review by sreeramsama — APPROVED** [review]

**Pull request merged**